modules/labs-react/toast/stories/toast.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Toast} from '@workday/canvas-kit-labs-react/toast';
+import {Basic} from './examples/Basic';
+import {ToastAlert} from './examples/ToastAlert';
+import {WithCloseButton} from './examples/WithCloseButton';
+import {ToastDialog} from './examples/ToastDialog';
+import {WithActionLinkAndCloseIcon} from './examples/WithActionLinkAndCloseIcon';
+import {WithPopper} from './examples/WithPopper';
+import {RTL} from './examples/RTL';
+
+<Meta title="Labs/Toast/React" component={Toast} />
+
+# Canvas Kit Toast
+
+`Toast` is a [compound component](/getting-started/for-developers/resources/compound-components/)
+that contains updates or messages about the status of an application's process.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react
+```
+
+## Usage
+
+### Basic Example
+
+`Toast` includes a container `Toast` component and the following subcomponents: `Toast.Body`,
+`Toast.CloseIcon`, `Toast.Icon`, `Toast.Message`, and `Toast.Link`.
+
+`Toast` supports different modes through the `mode` prop: `status`, `alert`, and `dialog`. Each mode
+conveys a different purpose of the message and assigns the necessary ARIA attributes to support that
+purpose and provide an accessible experience.
+
+By default, `mode` is set to `status`, which indicates the message contains advisory information
+such as a successful action.
+
+<ExampleCodeBlock code={Basic} />
+
+A `status` `Toast` will wrap the message within a `polite` ARIA live region with a `role` of
+`status`.
+
+Here's a more complete example with a button which triggers a dismissable `Toast`. The `Toast` is
+positioned using `Popper`.
+
+<ExampleCodeBlock code={WithPopper} />
+
+### Alert
+
+Set the `mode` prop to `alert` to convey urgent and important information such as an error.
+
+<ExampleCodeBlock code={ToastAlert} />
+
+An `alert` `Toast` will wrap the message within an `assertive` ARIA live region with a `role` of
+`alert`.
+
+### Dialog
+
+Set the `mode` prop to `dialog` to convey the presence of a follow-up action. You can also use the
+`useTostModel` to pass in a uniqueId:
+
+```tsx
+export const CustomIDToast = () => {
+  const model = useToastModel({
+    id: '12df5',
+    mode: 'dialog',
+  });
+  return (
+    <Toast model={model}>
+      <Toast.Icon icon={checkIcon} color={colors.greenApple400} />
+      <Toast.Body>
+        <Toast.Message>Your workbook was successfully processed.</Toast.Message>
+        <Toast.Link href="#href">View More Details</Toast.Link>
+      </Toast.Body>
+    </Toast>
+  );
+};
+```
+
+> **Note**: You must use `Toast.Message` subcomponent when the mode is `dialog` so that `id` is tied
+> to the message for accessibility
+
+<ExampleCodeBlock code={ToastDialog} />
+
+The `Toast` will be given a role of `dialog` along with an `aria-label` to indicate that it's a
+notification.
```

Comment:
```suggestion
### Dialog

Set the `mode` prop to `dialog` to convey the presence of a follow-up action.

<ExampleCodeBlock code={ToastDialog} />

The `Toast` will be given a role of `dialog` along with an `aria-label` to indicate that it's a
notification.

> **Note**: You must use the `Toast.Message` subcomponent when the `mode` is `dialog` so that `id`
> is tied to the message for accessibility.

You can also pass in a `model` with `useToastModel` to provide a specific `id` for the
`Toast.Message`:

```tsx
export const CustomIDToast = () => {
  const model = useToastModel({
    id: '12df5',
    mode: 'dialog',
  });
  return (
    <Toast model={model}>
      <Toast.Icon icon={checkIcon} color={colors.greenApple400} />
      <Toast.Body>
        <Toast.Message>Your workbook was successfully processed.</Toast.Message>
        <Toast.Link href="#href">View More Details</Toast.Link>
      </Toast.Body>
    </Toast>
  );
};
```
```